lib/google-apis/auth-client.js

```diff
@@ -83,18 +137,25 @@ AuthClient.prototype._getApplicationDefaultCredentials = function() {
 
     if (err) {
 
+      console.log('Stackdriver Error Reporting API client cannot initialize:\n',
+                  err, '\n',
+                  'Errors will not be reported to the Stackdriver API.');
+      self._cannotInit = true;
+      self._initError = err;
+      self.emit('init', self.getInitStatus());
+
       return;
     }
 
+    self._authClient = authClient;
+
     if (authClient.createScopedRequired && authClient.createScopedRequired()) {
 
       self._authClient = authClient.createScoped(scopes);
-    } else {
-
-      self._authClient = authClient;
     }
 
     self._hasInited = true;
+    self.emit('init', self.getInitStatus());
```

Comment:
You're guaranteed that `getInitStatus()` will return `true` here. I think you can get rid of `getInitStatus` unless there is some other need for it that I don't know of.